recipes/2dfatmic/build.xsh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env xonsh
+from xonsh.lib.os import indir
+
+mkdir -p build
+mkdir -p $PREFIX/bin
+with indir("build"):
+    for f in g`../*.f`:
+        if "main" in f:
+            main = f
+            continue
+        ![$FC -c @(f)]
+    ![$FC @(main) *.o -o 2dfatmic]
```

Comment:
Use FCFLAGS and LDFLAGS

Reply:
Aren't those applied automatically because they are in the environment?

Reply:
nm, they aren't